web_src/js/features/pull-view-file.js

```diff
@@ -53,9 +54,17 @@ export function initViewedCheckboxListenerFor() {
       const hasChangedLabel = form.parentNode.querySelector('.changed-since-last-review');
       hasChangedLabel?.remove();
 
+      const fileName = checkbox.getAttribute('name');
+
+      // check if the file is in our difftreestore and if we find it -> change the IsViewed status
+      const fileInPageData = diffTreeStore().files?.find((x) => x.Name === fileName);
```

Comment:
```suggestion
      const fileInPageData = diffTreeStore().files.find((x) => x.Name === fileName);
```

It's better to expose errors as early as possible if the code breaks.